tests/conftest.py

```diff
@@ -192,6 +196,9 @@ def run_slim_restart(cls, in_ts, recipe, out_dir, subpop_map=None, **kwargs):
         # on it, saving to files in out_dir.
         infile = os.path.join(out_dir, "in.trees")
         in_ts.dump(infile)
+        # for happy windows filepaths       
+        infile_str = infile.replace('\\', '\\\\')
+        print(infile_str)
```

Comment:
stray print

Reply:
actually those prints are useful! they only appear if you use `pytest -s`

happy to delete ofc

Reply:
okay got rid of this